snap/snapcraft.yaml

```diff
@@ -400,19 +413,7 @@ apps:
     command-chain:
       - bin/perl5lib-launch.sh
     plugs: [home, removable-media, network]
-  createuser:
-    adapter: full
-    command: usr/bin/createuser
-    environment:
-      LC_ALL: C.UTF-8
-      LANG: C.UTF-8
-      PGHOST: $SNAP_COMMON/sockets
-    command-chain:
-      - bin/perl5lib-launch.sh
-      # createuser should be run as the snap_daemon user, which is the user
-      # who is initially created as a role with postgres
-      - bin/drop
-    plugs: [home, removable-media, network]
+  # TODO: remove as part of secure postgres PR!
```

Comment:
what should be removed as part of the secure postgres PR?

Reply:
(and is that this PR?)

Reply:
leftover comment, will remove when I push my update